Source Packages/java/edu/slu/tpen/servlet/CreateProjectServlet.java

```diff
@@ -175,7 +172,7 @@ public String creatManuscriptFolioProject(HttpServletRequest request, HttpServle
                                     JSONObject image = images.getJSONObject(n);
                                     JSONObject resource = image.getJSONObject("resource");
                                     String imageName = resource.getString("@id");
-                                    int folioKey = textdisplay.Folio.createFolioRecordFromNewBerry(city, canvas.getString("label"), imageName, archive, m.getID(), 0); //why imageName.replace('_', '&')
+                                    int folioKey = textdisplay.Folio.createFolioRecordFromNewBerry(city, canvas.getString("label"), imageName, archive, -1, 0); //why imageName.replace('_', '&')
```

Comment:
I assume -1 doesn't break anything downstream that we've found. It is a good falsey value, but something like 987654 might be better.

Reply:
Not that I have seen.  I am not attached to `-1` and agree that if you aren't attached don't use it b/c it is just one of those things.